lib/compat.php

```diff
@@ -14,7 +14,7 @@
  * @return bool
  */
 function gutenberg_should_load_separate_block_assets() {
-	$load_separate_styles = gutenberg_supports_block_templates();
+	$load_separate_styles = gutenberg_is_fse_theme();
```

Comment:
So I understand, this will make it so that only themes with an `block-templates/index.html` will get the block style loading optimization?

Reply:
Yes :)